docker/broker/env/dev.env

```diff
@@ -0,0 +1,6 @@
+KAFKA_BROKER_ID=1
```

Comment:
We most likely won't need multiple environments so maybe start with `docker/broker/.env`?

Reply:
My issue with `.env` is that it is a hidden file. idk why that is the recommended approach..

Reply:
`docker.env`?

Reply:
SG